db/queries/categories.js

```diff
@@ -0,0 +1,74 @@
+const db = require('../connection');
+
+const getAllCategory = () => {
```

Comment:
Let's rename this to `getCategoriesForUser` since we're fetching the current user's categories.